public/store/privacy_policy.html

```diff
@@ -1,6 +1,5 @@
 <meta charset="utf-8" />
-<link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
-<link href="https://fonts.googleapis.com/css?family=Roboto+Slab" rel="stylesheet">
+<link rel="stylesheet" href="assets/google/fonts.css">
```

Comment:
why not be consistent throughout and use ` <link rel="stylesheet" href="{{ asset('store/assets/google/fonts.css') }}" >`?